devel/meson/Portfile

```diff
@@ -4,9 +4,9 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        mesonbuild meson 0.48.0
+github.setup        mesonbuild meson 0.48.1
 github.tarball_from releases
-revision            3
+revision            0
```

Comment:
Usually we don't specify `revision` for a revision 0. If you could delete this line, I can merge the update pretty quickly.

Reply:
Done